GitHubData/Models/ResourcesModel.cs

```diff
@@ -0,0 +1,17 @@
+// <copyright file="ResourcesModel.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace GitHubData.Models;
+
+/// <summary>
+/// Holds various resources information about the rate limits of
+/// an authorized or unauthorized account.
```

Comment:
Refactor code docs to below:

```suggestion
/// Holds information about the rate limits of an authorized or unauthorized request.
```